src/aleph/sdk/connectors/superfluid.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+import asyncio
+from decimal import Decimal
+from typing import TYPE_CHECKING, Optional
+
+from eth_utils import to_normalized_address, to_wei
+from superfluid import CFA_V1, Operation, Web3FlowInfo
+from web3 import Web3
+from web3.types import TxParams
+
+from aleph.sdk.conf import settings
+
+if TYPE_CHECKING:
+    from aleph.sdk.chains.ethereum import LocalAccount
+
+
+async def sign_and_send_transaction(
+    account: LocalAccount, tx_params: TxParams, rpc: str
+) -> str:
+    """
+    Signs and broadcasts a transaction using ETHAccount
+    @param tx_params - Transaction parameters
+    @param rpc - RPC URL
+    @returns - str - The transaction hash
+    """
+    web3 = Web3(Web3.HTTPProvider(rpc))
+
+    def sign_and_send():
+        signed_txn = account.sign_transaction(tx_params)
+        transaction_hash = web3.eth.send_raw_transaction(signed_txn.rawTransaction)
+        return transaction_hash.hex()
+
+    # Sending a transaction is done over HTTP(S) and implemented using a blocking
+    # API in `web3.eth`. This runs it in a non-blocking asyncio executor.
+    loop = asyncio.get_running_loop()
+    transaction_hash = await loop.run_in_executor(None, sign_and_send)
+    return transaction_hash.hex()
```

Comment:
Looks like the hash is `hex`-ed twice. Why did no test catch this ?
```suggestion
    return transaction_hash
```

Reply:
This is not resolved: no test has been added to prevent such issue in the future.